student/models.py

```diff
@@ -1,3 +1,30 @@
-# from django.db import models
+from django.db import models
+from django.conf import settings
 
-# Create your models here.
+
+class Gender(models.TextChoices):
```

Comment:
Given how loaded gender issues are theses days, why do we even need to know it? Let alone store it in the database?

Reply:
Will remove it..